app/translations/de/LC_MESSAGES/messages.po

```diff
@@ -284,8 +284,8 @@ msgid ""
 "This is an automatic email from %(app_name)s for organizers of the "
 "%(session_event_name)s."
 msgstr ""
-"Dies ist eine automatische E-Mail von %(app_name)s für die Organisatoren der "
-"Sitzungen %(session_event_name)s."
+"Dies ist eine automatische E-Mail von %(app_name)s für die Organisatoren "
+"der Sitzungen %(session_event_name)s."
```

Comment:
I saw that before and got confused already. Shouldn't that variable be called "session_name" instead of "session_event_name"?

Reply:
Earlier it was this only. Should I change this?? It will not matter what the name is, the correct value would be put there regardless.

Reply:
If it is easy to change would be great if you could change it to bring more logic to this.